workspace/spec/lita/handlers/teamcity_spec_anki.rb

```diff
@@ -297,6 +297,49 @@
     end
   end
 
+  describe '#list' do
+    build_type_json_str = '{"buildType":[{"id":"Wildcard_A","projectName":"Wildcard :: A"},
+                                         {"id":"B_Wildcard","projectName":"Wildcard :: B"},
+                                         {"id":"Test_ABC","projectName":"Test :: ABC"}]}'
+    before do
+      stub_request(:get, /buildTypes/).to_return(:status => 200, :body => build_type_json_str)
+    end
+
+    it 'replies the correct builds if wildcard exists' do
+      wildcard = "Wildcard"
+      expected_output = "Here are the builds I found:\n\n" \
+                        "Wildcard :: A :: id=`Wildcard_A`\n" \
+                        "Wildcard :: B :: id=`B_Wildcard` \n\n" \
+                        "Use a build `id` from the list to trigger a build. *@buildbot build `id`*"
+      send_command("list #{wildcard}*")
+      expect(replies.last).to eq(expected_output)
+    end
+
+    it 'replies all builds if wildcard is empty' do
```

Comment:
Because wildcard is boolean variabl, so I think summary in case should be:
<code>replies all builds in case having not wildcard</code>